.changeset/breezy-parrots-press.md

```diff
@@ -0,0 +1,7 @@
+---
+"@comet/admin": patch
+---
+
+Adapt styling of `DateTimePicker` on mobile devices to improve readability of the placeholder
+
+Return `null` for `ClearInputAdornment` if there is no content to be cleared. This avoids overlapping placeholders in the field.
```

Comment:
As this doesn't only apply to the component mentioned in the changeset's title, this should be a separate changeset imo. 

Also the technical implementation is irrelevant to the dev's reading the changelog so it should be enough to mention that the placeholder is no longer overlapped by a non-visible clear-button.